app/src/main/java/net/gsantner/opoc/util/GsContextUtils.java

```diff
@@ -1556,6 +1558,11 @@ public void draftEmail(final Context context, final String subject, final String
         showChooser(context, intent, null);
     }
 
+    private static File checkPath(final String path) {
+        final File f;
+        return (!TextUtils.isEmpty(path) && (f = new File(path)).canRead()) ? f : null;
+    }
```

Comment:
If it's required still and it's those ~4 lines, I think it would fit to FileUtils class